src/main/java/frc/robot/subsystems/Tower.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+// Subsystem that controls the tower of the robot
+
+public class Tower extends SubsystemBase {
+  /** Creates a new Tower. */
+  private final CANSparkMax towerMotor;
+
+  /**
+   * Constructs a Tower with a {@link CANSparkMax} at the given CAN ID.
+   *
+   * @param towerMotorId The CAN ID of the Tower motor
+   */
+
+  public Tower(int towerMotorId) {
+    this(towerMotorId, MotorType.kBrushless);
+    setBrakingMode(IdleMode.kBrake);
```

Comment:
Braking mode should be set in the main constructor. If you set it in this constructor, it will not be set if the other is called. Also, you should not expose a function to set the braking mode, so please call the method from the motor object directly.